lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -220,6 +221,14 @@ protected function transformCampaign($data) {
 
         $output['staff_pick'] = $data->staff_pick;
 
+        if ($data->competition == '1') {
+          $data->competition = TRUE;
+        }
+        elseif ($data->competition == '0') {
+          $data->competition = FALSE;
+        }
```

Comment:
You can keep this in the build (so it's available as a boolean outside the API, so for the DS app internal stuff) and just run it through this function: 

https://github.com/DoSomething/phoenix/blob/4da588c3ffd383111e8650100dfa4673f11f46a4/lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module#L127